custom-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,974 @@
+          {
+            "name": "query",
+            "in": "body",
+            "description": "Query Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/APITriggerQueryResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "APIEntityListResponse": {
+      "description": "List of Entities",
+      "type": "object",
+      "properties": {
+        "EntityName": {
+          "description": "Entity name as it is in database",
+          "type": "string"
+        },
+        "EntityNameDisplay": {
+          "description": "Entity display name",
+          "type": "string"
+        },
+        "AllowAdd": {
+          "description": "Indicates if allowed to add new record",
+          "type": "boolean"
+        },
+        "AllowEdit": {
+          "description": "Indicates if allowed to edit record(S)",
+          "type": "boolean"
+        },
+        "AllowView": {
+          "description": "Indicates if allowed to view record(s)",
+          "type": "boolean"
+        },
+        "EntityNameForAPI": {
+          "description": "Entity name that is used in CSS API",
+          "type": "string"
+        }
+      }
+    },
+    "CSSAPICustomMessage": {
+      "description": "",
+      "required": [
+        "LogId",
+        "Message"
+      ],
+      "type": "object",
+      "properties": {
+        "LogId": {
+          "description": "",
+          "type": "string"
+        },
+        "Message": {
+          "description": "",
+          "type": "string"
+        },
+        "Exception": {
+          "description": "",
+          "type": "object"
+        }
+      }
+    },
+    "APISchemaResponse": {
+      "description": "List of columns, data types, and any restrictions or flags (NULL, Length, Primary Key, etc.) in a table",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "type": "string"
+        },
+        "IsPrimaryKey": {
+          "type": "boolean"
+        },
+        "IsIdentity": {
+          "type": "boolean"
+        },
+        "IsNullable": {
+          "type": "boolean"
+        },
+        "DBType": {
+          "type": "string"
+        },
+        "MaxLength": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "DataPrecision": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "DataScale": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "HasDefaultValue": {
+          "type": "boolean"
+        },
+        "IsComputedColumn": {
+          "type": "boolean"
+        },
+        "IsPassword": {
+          "type": "boolean"
+        }
+      }
+    },
+    "APIQueryRequest": {
+      "description": "",
```

Comment:
@nizarhandal Empty property values can be removed until unless you have a specific reason to keep them.